ext/openssl/openssl.c

```diff
@@ -780,7 +777,7 @@ static int php_openssl_parse_config(struct php_x509_request * req, zval * option
 		zend_long cipher_algo = Z_LVAL_P(item);
 		const EVP_CIPHER* cipher = php_openssl_get_evp_cipher_from_algo(cipher_algo);
 		if (cipher == NULL) {
-			php_error_docref(NULL, E_WARNING, "Unknown cipher algorithm for private key.");
+			zend_value_error("Unknown cipher algorithm for private key");
```

Comment:
So this cipher is used only if there is a private key passphrase and it can be caused by invalid conflict but doesn't need to be an issue because passprhase is not always used or it can be just something wrong in the config. I'd keep warning in here rather than prevent the whole operation to minimize BC breaks.